markets/perps-market/contracts/storage/GlobalPerpsMarketConfiguration.sol

```diff
@@ -32,10 +34,9 @@ library GlobalPerpsMarketConfiguration {
          */
         mapping(address => uint256) referrerShare;
         /**
-         * @dev mapping of configured synthMarketId to max collateral amount.
-         * @dev USD token synth market id = 0
+         * @dev previously maxCollateralAmounts[collateralId] was used in storage slot
          */
-        mapping(uint128 => uint256) maxCollateralAmounts;
+        mapping(uint128 => uint256) __unused_1;
```

Comment:
moved to collateral configuration.  the function name for setting max collateral has changed which will force the value to be set again.